shared-module/keypad/__init__.c

```diff
@@ -58,8 +58,14 @@ void keypad_tick(void) {
 }
 
 void keypad_reset(void) {
```

Comment:
@dhalbert Couldn't we switch this to using a finalizer instead?

Reply:
Certainly, but is there other stuff first? Is the PR blocked if it's not a finalizer yet?

Reply:
If I could get some advice on implementing one of these finalizers that would be great.